src/Orleans/Messaging/SqlMembershipTable.cs

```diff
@@ -21,489 +21,148 @@ MIT License
 TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 */
 
+using Orleans.Messaging;
+using Orleans.Runtime.Configuration;
+using Orleans.Runtime.Storage.Relational;
 using System;
-using System.Net;
-using System.Text;
-using System.Data;
-using System.Data.SqlClient;
 using System.Collections.Generic;
 using System.Threading.Tasks;
 
-using Orleans.Messaging;
-using Orleans.Runtime.Configuration;
```

Comment:
<img src="http://i.stack.imgur.com/71HGg.png" alt="sort usings with system first" />

Please check this option and resort usings of each code file.